docs/translation.md

```diff
@@ -5,63 +5,107 @@ This document provides guidelines for contributing translations and getting star
+
+**Tasks of the translator:**
 
-Keycloak already supports a lot of translations. 
-While were in the transitioning process towards Weblate, the following translations are available in Weblate:
+1. Optional: Create an issue or discussion as outlined in the [contributing docs](../CONTRIBUTING.md).
+2. Update the `messages_*.properties` files for one or more languages.
+3. Commit the changes with sign-off as outlined in the [contributing docs](../CONTRIBUTING.md). If you created an issue in the first step, reference it in the commit. 
+4. Create a pull request. If you created an issue in the first step, link it to the issue as described in [contributing docs](../CONTRIBUTING.md).
+5. Find a reviewer that is a native speaker in that language. 
 
-- German
-- Dutch
-- Japanese
-- Catalan
-- Spanish
+**Task of the reviewer:** 
 
-For all other translations, look for the `messages_*.properties` files in the main repository and create a pull request with your translations. 
+1. As a native speaker, review the changes in the pull request. Once you are satisfied with the changes, leave a comment in the pull request that you approve the changes.
 
-Please note that while we aspire to support more translations in the future, our current focus is on these translations. We rely on volunteers like you to take the initiative in steering the translation efforts for additional translations. If you're interested in contributing translations for a language not listed above, add a comment to the [localization platform discussion](https://github.com/keycloak/keycloak/discussions/9270).
+**Tasks of the maintainer:** 
 
-If you have any questions or need assistance, feel free to reach out to the language maintainers listed below:
+1. Once the review from the native speaker is in, check that the changes to the `messages_*.properties` files are syntactically correct.
+2. If there are any translation pull requests created by Weblate, assure to merge them first to avoid conflicts in Weblate which are difficult to resolve.
+3. If the changes are ok, approve the PR and merge it. If there is an existing issue, reference that in the squash-message, otherwise reference the issue of the pull request.
 
-* German: [Robin Meese](https://github.com/robson90)
+## Using Weblate to to update translations
+
+Keycloak uses [Weblate](https://hosted.weblate.org/projects/keycloak/), a web-based translation platform, to manage translations of user interfaces and error messages.
+
+It allows for notifications when the original string changes, and keeps track of missing translations. It also allows contributors without knowledge of Git to contribute to the translations.
+
+The following translations are available in Weblate. If you have any questions or need assistance, feel free to reach out to the language maintainers listed below:
+
+* German: [Robin Meese](https://github.com/robson90) && [Alexander Schwartz](https://github.com/ahus1)
```

Comment:
Nit: let's get rid of the double ampersands here, a single one will suffice for English.